### ISSUE TITLE
Superfluous quotes in lockfile

### ISSUE THREAD

**Issue by jonashaag** [prefix-dev/pixi#247]
```
metadata:
  content_hash:
    osx-arm64: '"e90c2ee71ad70fc0a1c8302029533a7d1498f2bffcd0eaa8d2934700e775dc1d"'
```

Generated with Pixi 0.0.8

**Comment by pavelzw** [comment]
It may also make sense to add a version to the lockfile similar to how pnpm does it. https://github.com/mamba-org/setup-micromamba/blob/2ee1938ba9ff3b184b94f8bdd4a0b0cd9976bbe9/pnpm-lock.yaml#L1
NPM does it as well in their `package-lock.json` https://github.com/puppeteer/puppeteer/blob/9a814a365644a1e404eb36266a08f579bb2c26ac/package-lock.json#L3

**Comment by baszalmstra** [comment]
Thanks @jonashaag ! I fixed the issue in [rattler](https://github.com/mamba-org/rattler/pull/277). Should be included in the next release of pixi. 

@pavelzw We implemented the conda-lock file format which also doesn't include a version. 

This is a bit off-topic but pixi also doesn't use the `content-hash` at all, there are better ways to solve the problems the content-hash was meant to solve. It is also already possible with the current format. That is what we are using in pixi.
We propose removing the field altogether, see https://github.com/conda/conda-lock/issues/432.
